docs/ActiveLabel.rst

```diff
@@ -1,135 +1,320 @@
+example movie database from above, you could create an ``ActiveLabel`` module which only adds Actor methods and
+properties to an instance of Person if a Person node also has an Actor label. Or only adds InShowbusiness methods
+and properties to an instance of Person if a Person node has `either` Actor or Director labels.
+
+``ActiveLabel`` can fully replace ``ActiveNode`` inheritence, but it involves a different way of thinking
+then what many ActiveRecord developers might be used to. If you're just starting out with Neo4jrb, you might find
+it easiest to stick with the "ActiveRecord" like workflow provided by ``ActiveNode`` and inheritence.
+But as you get more comfortable with Neo4j's flexibility and 
+polymorphism, you'll might find that ``ActiveLabel`` is the better option for many tasks.
 
 .. code-block:: ruby
 
   class Person
     include Neo4j::ActiveNode
+    include Actor
+    include Director
+    include User
 
     property :name, type: String
-
-    label :HasAddress
-    label :Destroyable
   end
 
-  class Organization
+  class Movie
     include Neo4j::ActiveNode
 
-    property :title, type: String
+    property :name, type: String
+  end
 
-    label :HasAddress
-    label :Destroyable
+.. code-block:: ruby
+
+  module Actor
+    include Neo4j::ActiveLabel
+    include InShowbusiness
+
+    has_many :out, :acts_in, type: :ACTS_IN, model_class: :Movie
   end
 
+  module Director
+    include Neo4j::ActiveLabel
+    include InShowbusiness
 
-.. code-block:: ruby
+    has_many :out, :directed, type: :DIRECTED, model_class: :Movie
+  end
+
+  module InShowbusiness
+    include Neo4j::ActiveLabel
 
-  class Address
-    property :line1, type: String
-    property :line2, type: String
-    property :country, type: String
-    property :postal_code, type: String
+    self.associated_labels = [:Actor, :Director]
+    self.associated_labels_matcher = :any
+
+    property :biography
+    property :lastModified
+    property :version
   end
 
-  module HasAddress
+  module User
     include Neo4j::ActiveLabel
 
-    included do
-      has_one :out, :address, type: :HAS_ADDRESS
-    end
+    property :login
+    property :password
+    property :roles
 
     module InstanceMethods
-      def distance_from(has_address_object)
-        address.distance_from(has_address_object.address)
+      def administrate_stuff
+        puts "administrate stuff"
       end
     end
   end
 
 
+Creating
+--------
+
+``ActiveLabel`` modules are defined by creating a standard ruby module with ``include Neo4j::ActiveLabel``.
+By convention, the ``ActiveLabel`` module will be associated with a label equal to the module name. For example,
+in the example above, the ``Actor`` ``ActiveLabel`` module is associated with the ``:Actor`` label. You can 
+customize the label(s) which an ``ActiveLabel`` module is associated with using ``self.associated_labels =``. You must also
```

Comment:
This may just come down to personal preference. I originally wrote 

>in the example above, the ``Actor`` ``ActiveLabel`` module follows the ``:Actor`` label. You can customize the label(s) which an ``ActiveLabel`` module follows using ``self.associated_labels =``.

But I found the `follows` metaphor to be less intuitive for me than the `associated` metaphor, which resulted in the change to `self.associated_labels =`.

I don't feel strongly about this though.